CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 1.9.0
+
+* New project generator updates
+  * bump NervesPack to 0.4 which drops `:nerves_firwmare_ssh` in
+    favor of `:nerves_ssh` and `:ssh_subsystem_fwup` for access and updates.
+  * bump Nerves to 1.6.3
+  * bump NervesRuntime to 0.11
+  * bump RingLogger to 0.8
+
+* Enhancements
+  * Start support for Elixir 1.11
+
```

Comment:
```suggestion
```

I'm for deleting this for now since the fixes were for things that were also issues in Elixir 1.10 and Elixir 1.11 doesn't work yet.